dashboard/app/admin.go

```diff
@@ -212,9 +214,89 @@ func updateBugBatch(c context.Context, keys []*db.Key, transform func(bug *Bug))
 	return nil
 }
 
+// TODO: move this functionality to some separate HTML page.
+func collectStats(c context.Context, w http.ResponseWriter, r *http.Request) error {
```

Comment:
Maybe add some http endpoint for this?
It's restricted to admin, so there's no downside. Otherwise it's not possible to use it w/o redeploying, right?

Reply:
Done.